process/block/baseProcess.go

```diff
@@ -981,6 +997,11 @@ func (bp *baseProcessor) saveMetaHeader(header data.HeaderHandler, headerHash []
 	if errNotCritical != nil {
 		log.Warn("saveMetaHeader.Put -> MetaBlockUnit", "error", errNotCritical.Error())
 	}
+
+	elapsedTime := time.Since(startTime)
+	if elapsedTime >= time.Second {
```

Comment:
add constant for time.second - like warn time - do the same for trie-snapshot - commit

Reply:
Added constant. Actually trie-snapshot action is done on go routines so measuring them are not useful.

```
func (adb *AccountsDB) SnapshotState(rootHash []byte) {
	adb.mutOp.Lock()
	defer adb.mutOp.Unlock()

	log.Trace("accountsDB.SnapshotState", "root hash", rootHash)
	adb.mainTrie.EnterSnapshotMode()

	go func() {
		adb.mainTrie.TakeSnapshot(rootHash)
		adb.snapshotUserAccountDataTrie(rootHash)
		adb.mainTrie.ExitSnapshotMode()
	}()
}

// SetStateCheckpoint sets a checkpoint for the state trie
func (adb *AccountsDB) SetStateCheckpoint(rootHash []byte) {
	adb.mutOp.Lock()
	defer adb.mutOp.Unlock()

	log.Trace("accountsDB.SetStateCheckpoint", "root hash", rootHash)
	adb.mainTrie.EnterSnapshotMode()

	go func() {
		adb.mainTrie.SetCheckpoint(rootHash)
		adb.snapshotUserAccountDataTrie(rootHash)
		adb.mainTrie.ExitSnapshotMode()
	}()
}

```

Reply:
Now, all the store.Put actions from baseProcess, shardBlock, metaBlock are measured